### PR TITLE
Added Fastlane changelogs

### PR DESCRIPTION
**Pull request by CommanderStorm**
## Issue

This fixes the following issue(s):
- Resolves: #1351

## Screenshot
N.A.

## Why this is useful for all students
A student requested this, so it is apparently useful. I am open for wording/…
Should I translate those changelogs into English as well?

### CODE REVIEW

**Comment by Bentipa** [comment]
Do we show them inside of the app or where are they used?

**Comment by CommanderStorm** [comment]
> Do we show them inside of the app or where are they used?

They are from what I can tell used in the App store. At least that is what the Issue lead me to believe...

File: fastlane/metadata/android/de-DE/changelogs/523.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+v2.0
+
+- Neues Design im freshen look - lasst uns wissen wie es euch gefällt => app@tum.de
+- Verbessertes Benachrichtigungskonzept. Eventuell zeigen wir euch jetzt zu besseren Uhrzeiten den Speiseplan für die Mensa an.
+- Featuring Uhrenturm im Menü, weil der einfach besser ist
+- Logout ist nun über die Einstellungen Möglich
```
Comment (tobiasjungmann) [inline fastlane/metadata/android/de-DE/changelogs/523.txt:6]:
```suggestion
- Logout ist nun über die Einstellungen möglich.
```

File: fastlane/metadata/android/de-DE/changelogs/527.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+v2.4
+
+- The Event Details now consistently shows the buy button
+- Added the ability to dismiss the beta info
```
Comment (tobiasjungmann) [inline fastlane/metadata/android/de-DE/changelogs/527.txt:4]:
```suggestion
- In den Veranstaltungsdetails wird nun die Schaltfläche "Kaufen" permanent angezeigt.
- Es wurde die Möglichkeit hinzugefügt, die Beta-Informationen ignorieren zu können.
```

File: fastlane/metadata/android/de-DE/changelogs/540.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+v3.12
+
+* Mehrere Bugfixes:
+  * Version 3.11 wurde geskippt
+  * Verschiedeene Darkmode fixes
```
Comment (tobiasjungmann) [inline fastlane/metadata/android/de-DE/changelogs/540.txt:5]:
```suggestion
  * Verschiedene Darkmode Fixes
```

File: fastlane/metadata/android/de-DE/changelogs/539.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+v3.10
+
+* Mehrere Bugfixes:
+  * Die Notenansicht enthält nun den Hinweis, dass der Angezeige Durchschnitt nicht der akkurate Durchschnitt ist
```
Comment (tobiasjungmann) [inline fastlane/metadata/android/de-DE/changelogs/539.txt:4]:
```suggestion
  * Die Notenansicht enthält nun den Hinweis, dass der angezeige Durchschnitt nicht zwingend dem tatsächlichen Durchschnitt entspricht
```

File: fastlane/metadata/android/de-DE/changelogs/542.txt [hunk 5]
```diff
@@ -0,0 +1,5 @@
+v3.14
+
+* Mehrere Bugfixes:
+  - Wenn die bevorzugte Haltestelle für einen Campus-Standort geändert wird, wird die Änderung nun auch in der MVVCard, die auf dem Startbildschirm der App angezeigt wird, berücksichtigt.
+  - Der Kalendar handelt nun Zeitzohnenwechsel so wie man es erwarten würde.
```
Comment (tobiasjungmann) [inline fastlane/metadata/android/de-DE/changelogs/542.txt:5]:
```suggestion
  - Der Kalendar reagiert nun auf Zeitzonenwechsel so wie man es erwarten würde.
```

**Review by tobiasjungmann — COMMENTED** [review]
Now I am also able to review comments instead of actual code :) Looks good from my side, I just added some very minor changes.

**Review by tobiasjungmann — APPROVED** [review]

**Comment by IzzySoft** [comment]
> Do we show them inside of the app or where are they used?

The one matching the latest release is shown along with the app description e.g. in my repo (and to my knowledge also on Play if you use Fastlane Supply) **if it exists** – which currently is not the case (that would be `567.txt` as of this writing). For an example, [see here](https://apt.izzysoft.de/fdroid/index/apk/com.github.frimtec.android.securesmsproxy); it would also be displayed inside the F-Droid client. Your app is currently missing this, as there's no changelog matching the latest version.

**Pull request merged**